deployer/templates/heapster.yaml

```diff
@@ -83,9 +82,9 @@ objects:
           - "--wrapper.username_file=/hawkular-account/hawkular-metrics.username"
           - "--wrapper.password_file=/hawkular-account/hawkular-metrics.password"
           - "--wrapper.allowed_users_file=/secrets/heapster.allowed-users"
-          - "--wrapper.endpoint_check=https://hawkular-metrics:443/hawkular/metrics/status"
+          - "--wrapper.endpoint_check=https://hawkular-metrics.openshift-infra.svc.cluster.local:443/hawkular/metrics/status"
```

Comment:
The downward api can be used to request information, such as which namespace your project is deployed tohttps://kubernetes.io/docs/tasks/configure-pod-container/downward-api-volume-expose-pod-information/

See: https://github.com/openshift/origin-metrics/blob/master/deployer/templates/hawkular-metrics.yaml#L106-L109

You just need to add that to your template and use the envar here